server/data/licenceClientTypes.ts

```diff
@@ -1,6 +1,8 @@
 import { Licence, LicenceStage } from './licenceTypes'
 
-export type ConditionVersion = 1 | 2
+export type AdditionalConditionsVersion = 1 | 2
+
+export type StandardConditionsVersion = 1
```

Comment:
Set this as just 1 for now. We'll add version 2 in the next ticket to implement the usage of these. Not entirely sure yet if the default/historical will be 1 or null.

Reply:
some historical will require a migration once we know how to identify those